### PR TITLE
Adds Code of Conduct to Alliance

### PR DESCRIPTION
**Pull request by coyotesqrl**
#### What does this PR do?
Adds CoC and moves other repository meta-files into .github directory

#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)
@blen-desta @ahoffer 

#### Choose 2 committers to review/merge the PR.
(please choose ONLY two committers from below, delete the rest)
@rzwiefel
@shaundmorris

#### How should this be tested?
No testing required; these are repo metafiles.

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: .github/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at [ddf@codice.org](mailto:ddf@codice.org). All
```
Comment (coyotesqrl) [inline .github/CODE_OF_CONDUCT.md]: This line has the only change from the source CoC and is also the one that we may want to discuss. It seems reasonable to me to use the ddf@codice.org mailbox for community complaints for Alliance as well as DDF, but others may differ and think we need a separate mailbox. I don't feel strongly either way.

Comment (ahoffer) [reply .github/CODE_OF_CONDUCT.md]: They are both Codice projects and it makes sense that all Codice project have the same code of conduct. Maybe it should be codice@codice.org?

Comment (coyotesqrl) [reply .github/CODE_OF_CONDUCT.md]: I'm pretty indifferent. Hoping @shaundmorris has a strong opinion because I'm adopting that one.

Comment (shaundmorris) [reply .github/CODE_OF_CONDUCT.md]: I like keeping it the same, since Alliance is considered a sub-project of DDF.

**Pull request merged**